### ISSUE TITLE
Installing helm chart fails to "not found"

### ISSUE THREAD

**Issue by jaakkom** [p2-inc/helm-charts#5]

**Comment by darknos** [comment]
I got the same issue, please help to resolve

**Comment by jaakkom** [comment]
Created fork to older working version: https://github.com/Fisplay/helm-charts

**Comment by xgp** [comment]
Hi @jaakkom and @darknos . Thanks for raising this. Sincere apologies on this. We are in the middle of reevaluating owning our own chart rather than using the Bitnami or Codecentric charts with example values files for using the Phase Two extensions. We don't currently have the bandwidth to update these, so we are trying to find the best solution. 

Our current leaning is to use the Codecentric keycloakx chart https://github.com/codecentric/helm-charts/tree/master/charts/keycloakx with the `image.repository` var set to `quay.io/phasetwo/phasetwo-keycloak` and the `args` var set to `--spi-email-template-provider=freemarker-plus-mustache --spi-email-template-freemarker-plus-mustache-enabled=true --spi-theme-cache-themes=false` (as a starting point). We really like the Codecentric folks, and believe they have built the best chart out there.

**Comment by xgp** [comment]
@vitas-pm Yes. We're aware that there isn't a great choice, but we think the Codecentric chart is better than the Bitnami chart. The Bitnami chart makes a lot of assumptions about using their image. We wish we had the resources at this point to maintain an excellent chart for Keycloak that can use our or the default image, but we just don't have the customer demand or the time to do it right now.
